docs/Makefile.am

```diff
@@ -35,8 +35,8 @@ HTMLPAGES = $(GENHTMLPAGES) index.html
 
 # Build targets in this file (.) before cmdline-opts to ensure that
 # the curl.1 rule below runs first
-SUBDIRS = libcurl . cmdline-opts
-DIST_SUBDIRS = $(SUBDIRS) examples
+SUBDIRS = cmdline-opts
```

Comment:
I think this change conflicts with the comment above. As far as I understand the comment and the `$(abs_builddir)/curl.1:` rule in the same file, we need . before cmdline-opts.